backend/app/api/admin_routes/stats.py

```diff
@@ -31,3 +37,37 @@ def chat_origin_trend(
 ) -> ChatStats:
     stats = chat_repo.chat_trend_by_origin(session, start_date, end_date)
     return ChatStats(start_date=start_date, end_date=end_date, values=stats)
+
+class ChatOrigin(UUIDBaseModel):
+    origin: str
+
+@router.get("/admin/stats/chats/origins")
+def list_chat_origins(
+    session: SessionDep,
+    user: CurrentSuperuserDep,
+    params: Params = Depends(),
+) -> Page[ChatOrigin]:
+    return paginate(
```

Comment:
BTW, I perfer to using `ChatRepo` to handle all of DB access operaction of `Chat` model, so that we can reuse it in other place.